projects/core/src/datetimepicker/datetimepicker-input.ts

```diff
@@ -22,8 +22,8 @@ import {
   ValidatorFn,
   Validators,
 } from '@angular/forms';
-import { MAT_LEGACY_INPUT_VALUE_ACCESSOR as MAT_INPUT_VALUE_ACCESSOR } from '@angular/material/legacy-input';
```

Comment:
what about the constant `MAT_LEGACY_INPUT_VALUE_ACCESSOR `?

Reply:
Dawn, I miss one, I'll push fix soon

Reply:
hmm I see, sorry. I did misread that too :D